docs/Building-Against-Non-Homebrew-Dependencies.md

```diff
@@ -0,0 +1,11 @@
+# Building-Against-Non-Homebrew-Dependencies.md
```

Comment:
You probably want the title here, rather than the filename.

Reply:
Thanks @EricFromCanada!